modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements/evidence_submissions_controller.rb

```diff
@@ -39,7 +39,7 @@ def set_notice_of_disagreement
 
         def set_submission_attributes
           @submission_attributes ||= {
-            source: params['headers']['X-Consumer-Username'],
+            source: request.headers['X-Consumer-Username'],
```

Comment:
I'm getting nil for `request.headers` when I try this locally.

Reply:
I think the dot notation and the parameter notation both work for the request object.

Reply:
We are using dot notation elsewhere in our controllers so I think we can stick with that.